curriculum/challenges/english/07-scientific-computing-with-python/learn-encapsulation-by-building-a-projectile-trajectory-calculator/663353465bfb14259717da93.md

```diff
@@ -0,0 +1,388 @@
+---
+id: 663353465bfb14259717da93
+title: Step 13
+challengeType: 20
+dashedName: step-13
+---
+
+# --description--
+
+Create a method `create_coordinates_table` that uses the `__coordinates` property and visualizes the coordinates, where x and y have always the same position, the values for x are always integers, and the values for y are always rounded to two decimal places:
+
+```py
+  x      y
+  1   0.10
+  2   0.20
+  3   0.30
+  4   0.40  
+```
+
```

Comment:
This step it's a bit challenging if we don't explain how to use the [format specification mini-language](https://docs.python.org/3/library/string.html#format-specification-mini-language) before it.

It's okay to count spaces, but using round() you don't get always the same number of decimal digits because `1.10` will be displayed as `1.1`.
We should consider that.